src/main/java/core/basesyntax/model/Message.java

```diff
@@ -1,8 +1,22 @@
 package core.basesyntax.model;
 
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "messages")
 public class Message {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String content;
+    @OneToOne(cascade = {CascadeType.PERSIST, CascadeType.REMOVE}, fetch = FetchType.EAGER)
```

Comment:
`OneToOne` and `ManyToOne` default `FetchType` is `EAGER` so this one is redundant
```suggestion
    @OneToOne(cascade = {CascadeType.PERSIST, CascadeType.REMOVE})
```